docs/design/3_2_multirotor_v2.tex

```diff
@@ -1,26 +1,231 @@
+
+A dual-blade helicopter, for example, requires fewest number of motors. But like a single-blade helicopter, the system in control theory sense, is naturally unstable and requires difficulty to stabilize the system. A tricopter has odd number of motors which also shares similar characteristics with a helicopter where the system requires an additional tail motor to compensate for the uneven torque. 
+
+Configurations using more than 4 motors such as a hexacopter or an octocopter are very expensive for the amount of parts they require. Drones of these configuration classes are very stable with failure redundancy. However, these drones are typically much larger (0.6m to 1.5m wide) and much heavier (up to 25 kg). As a result, they are typically used in industrial, filming, and agriculture applications with a typical unit cost of at least several thousand dollars. These configurations are not suitable for our applications.
+
 \paragraph{Airframe}
 
+\begin{figure}[h]
+	\centering
+	\begin{subfigure}[b]{0.33\textwidth}
+		\centering
+		\includegraphics[width=\textwidth]{img/djiphantom4}
+		\caption{DJI Phantom 4 (350 mm)}
+		\label{fig:djiphantom}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.33\textwidth}
+		\centering
+		\includegraphics[width=\textwidth]{img/f450frame}
+		\caption{Generic DIY frame kit (450 mm)}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.33\textwidth}
+		\centering
+		\includegraphics[width=\textwidth]{img/jmt560}
+		\caption{JMT X4 Carbon Fiber Frame (560 mm)}
+		\label{fig:jmtx4}
+	\end{subfigure}
+	~
+	\begin{subfigure}[b]{0.33\textwidth}
+		\centering
+		\includegraphics[width=\textwidth]{img/djis1000}
+		\caption{DJI Spreadwings S1000 (1000 mm)}
+		\label{fig:djis1000}
+	\end{subfigure}
+	
+	\caption{Air Frame Options with Various Size Tiers }
+\end{figure}
+
+Multirotor airframe sizes are commonly designated by their \textit{motor-to-motor-diagonal-span} (MMDS), which measures the distance between the pair of motors which are the furthest apart. 
+
+We adopt a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fibre to enhance structural rigidity while keeping the system relatively lightweight. The frame features foldable arms and landing gears for portability, but we consider replacing the hinges with static parts to reduce  weight and achieve a longer flight time. The X4 560 mm frame is also wide enough to support propellers up to 14 inches in diameter. The carbon fibre material is light, strong, and rigid. Although this means that the items mounted on the frame would suffer from lack of dampening of vibrations, we implement compensating dampers for sensors separately.
```

Comment:
"we consider replacing the hinges"

Did we replace hinges in the end?

Reply:
Nope, I will rephrase this to suggest possible modification